certified-connectors/Nitro/apiDefinition.swagger.json

```diff
@@ -240,23 +240,136 @@
     "/webhooks": {
       "x-ms-notification-content": {
         "schema": {
+          "type": "object",
+          "required": [
+            "document"
+          ],
           "properties": {
             "document": {
+              "type": "string",
               "format": "byte",
-              "title": "Signed document",
-              "type": "string"
+              "title": "Signed document"
+            }
+          }
+        }
+      },
+      "post": {
+        "deprecated": true,
+        "x-ms-api-annotation": {
+          "revision": 1
+        },
+        "description": "Creates a webhook to notify signed document events",
+        "summary": "Triggers when a document is signed",
+        "operationId": "webhookDocumentSignedTriggetr",
+        "x-ms-trigger": "single",
+        "parameters": [
+          {
+            "name": "request",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "required": [
+                "endpoint",
+                "event"
+              ],
+              "properties": {
+                "endpoint": {
+                  "type": "string",
+                  "x-ms-notification-url": true,
+                  "x-ms-visibility": "internal"
+                },
+                "event": {
+                  "type": "string",
+                  "x-ms-visibility": "internal",
+                  "default": "esign.request.completed"
+                }
+              }
             }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created"
           },
+          "default": {
```

Comment:
@diegobernardes We would recommend you to replace the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.

Reply:
I'm not changing this method section. The only thing that I did is to mark it as deprecated. Git is showing the diff because I did duplicate it to introduce a new method `/v2/webhooks`.

Reply:
Default response with no schema definition would not make sense, Can you please add a schema to it

Reply:
> @diegobernardes We would recommend you to replace the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.

Ok, I changed the default to the expected status code.